recipes/zenpy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "zenpy" %}
+{% set version = "2.0.24" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zenpy-{{ version }}.tar.gz
+  sha256: 4fb01f8e7f5a9bcf33a61546804953ec9dc07625c9801b30f13b5fad9370429a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - cachetools >=3.1.0
+    - python-dateutil >=2.7.5
+    - pytz >=2018.9
+    - requests >=2.14.2
+    - six >=1.14.0
+
+test:
+  imports:
+    - zenpy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/facetoe/zenpy
+  summary: Python wrapper for the Zendesk API
+  license: GPL-3.0-only
```

Comment:
FYI: https://github.com/facetoe/zenpy/pull/504

This should include the license file in the next release.